detect-configuration/src/main/java/com/blackducksoftware/integration/hub/detect/configuration/DetectProperty.java

```diff
@@ -281,8 +282,8 @@
 
     @HelpGroup(primary = GROUP_PATHS, additional = { GROUP_BOMTOOL, SEARCH_GROUP_SEARCH })
     @HelpDescription("The tool priority for project name and version. The first tool in this list that provides a project name and version will be used.")
-    @AcceptableValues(value = { "DETECTOR", "DOCKER" }, caseSensitive = true, strict = true, isCommaSeparatedList = true)
-    DETECT_PROJECT_TOOL("detect.project.tool", "5.0.0", PropertyType.STRING, PropertyAuthority.None, "DETECTOR,DOCKER"),
+    @AcceptableValues(value = { "DETECTOR", "DOCKER", "BAZEL" }, caseSensitive = true, strict = true, isCommaSeparatedList = true)
+    DETECT_PROJECT_TOOL("detect.project.tool", "5.0.0", PropertyType.STRING, PropertyAuthority.None, "DOCKER,DETECTOR,BAZEL"),
```

Comment:
Was the docker and detector order deliberately swapped?

Reply:
Yes, I was making the ordering in the help match the default ordering. Thought it might avoid some potential confusion.